<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPage</class>
 <widget class="QWidget" name="AdminPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="backgroundAp" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QWidget" name="containAp" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="btnBoxAp">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#btnBoxAp{

	border: none;
	border-bottom: 2px solid #F5F6F8;
}


</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="checkBtnAp">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>60</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#checkBtnAp{
	background-color: #FFFFFF;
	font-family: '微软雅黑';
	font-size: 14px;
	font-weight: bold;
	color: #3ECEFF;
	border: none;
	border-bottom: 2px solid #3ECEFF;
}

</string>
              </property>
              <property name="text">
               <string>审核管理</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="roleBtnAp">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>60</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#roleBtnAp{
	background-color: #FFFFFF;
	font-family: '微软雅黑';
	font-size: 14px;
	font-weight: bold;
	color: #3ECEFF;
	border: none;
}
</string>
              </property>
              <property name="text">
               <string>人员管理</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1067</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="pageBoxAp" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <widget class="CheckPageForAp" name="checkAp" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1270</width>
              <height>686</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">/*
#checkAp{
	backgroud-color: #FFFFFF;
	font-size: 14px;
	font-family: '微软雅黑';
	color: #666666;
	border: none;
	border-bottom:2px solid #F5F6F8;
	
}
*/</string>
            </property>
           </widget>
           <widget class="RolePageForAp" name="roleAp" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1270</width>
              <height>686</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">/*
#roleAp{
	backgroud-color: #FFFFFF;
	font-size: 14px;
	font-family: '微软雅黑';
	color: #666666;
	border: none;
	border-bottom:2px solid #F5F6F8;
	
}

*/</string>
            </property>
           </widget>
           <zorder>roleAp</zorder>
           <zorder>checkAp</zorder>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckPageForAp</class>
   <extends>QWidget</extends>
   <header>checkpageforap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RolePageForAp</class>
   <extends>QWidget</extends>
   <header location="global">rolepageforap.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
